functional-tests/steps/then.js

```diff
@@ -22,3 +22,13 @@ Then(/^I see "([^"]*)" in the autocomplete suggestions$/, (text) => {
 	waitFor(".autocomplete__option");
 	checkContainsText("element", ".autocomplete__menu", null, text);
 });
+
+Then(
+	/^I should see a meta tag named "([^"]*)" with the content "([^"]*)"$/,
+	(name, content) => {
+		var selector = "meta[name='" + name + "']";
```

Comment:
Probably easier to read being a template literal here rather than concatenation and should be a const:
```suggestion
		const selector = `meta[name='${name}]`;
```